.env.example

```
@@ -1,9 +1,12 @@
 POSTGRES_DB=p2p_shopping
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
+
 JWT_SECRET=your-secret-key-here-at-least-32-characters-long
+GEMINI_API_KEY=your-gemini-api-key
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GEMINI_API_KEY key should go before the JWT_SECRET key

(UnorderedKey)